lxd/migration/utils.go

```diff
@@ -19,6 +19,9 @@ const BTRFSFeatureSubvolumeUUIDs = "header_subvolume_uuids"
 // ZFSFeatureMigrationHeader indicates a migration header will be sent/recv in data channel after index header.
 const ZFSFeatureMigrationHeader = "migration_header"
 
+// ZFSFeatureZvols indicates migration can send/recv zvols.
+const ZFSFeatureZvols = "header_zvols"
```

Comment:
Hrm, this isn't really what its indicating (as LXD can already do that), rather its indicating that it can handle sending/receiving *filesystem* zvols. Can we change to `ZFSFeatureZvolFilesystems = "header_zvol_filesystems"`?